### ISSUE TITLE
Add a way to delete broadcasts

### ISSUE THREAD

**Issue by WorldLanguages** [ScratchAddons/ScratchAddons#5420]
### Why this would be helpful/solve a problem

Self explanatory. This is an usual suggestion by users

### What this change does

.

### Demonstration(s) and additional context

I feel this might be a duplicate, or possibly some addon already adds a weird way to delete broadcasts, please confirm

**Comment by CubesterYT** [comment]
Nope, no option to delete broadcasts.
![image](https://user-images.githubusercontent.com/78769806/207154923-7e516c82-d677-43b2-8bae-4c166e8dbbb3.png)

**Comment by WorldLanguages** [comment]
Does Scratch automatically delete broadcasts that aren't used anymore in the project, after saving and reloading?

**Comment by Samq64** [comment]
Yes

**Comment by WorldLanguages** [comment]
> Yes, and when switching sprites.

I assume many users that request this feature do not know that.

**Comment by BroJac5246** [comment]
> > Yes, and when switching sprites.
> 
> I assume many users that request this feature do not know that.

That's what I rely on, it's pretty obvious when a broadcast is no longer in the dropdown.

**Comment by WorldLanguages** [comment]
I don't think this suggestion makes a lot of sense then. We already offer ways to find out where broadcasts are used, so users should use them to remove all instances of the broadcast, and Scratch will take care of it automatically.

**Comment by CubesterYT** [comment]
Maybe we could add a feature to mass replace a broadcast in use with another one, like the feature available in the DevTools addon in SA. 
![image](https://user-images.githubusercontent.com/78769806/207717646-028b00bf-f25b-425a-92c8-994b26d03798.png)
So that it can be easier to switch, also allowing for the broadcast to be deleted immediately.

**Comment by BroJac5246** [comment]
> I don't think this suggestion makes a lot of sense then. We already offer ways to find out where broadcasts are used, so users should use them to remove all instances of the broadcast, and Scratch will take care of it automatically.

While I wouldn't use it, this could still be potentially useful for users who are not aware that broadcasts are automatically deleted.

**Comment by penguinmoose** [comment]
> I don't think this suggestion makes a lot of sense then. We already offer ways to find out where broadcasts are used, so users should use them to remove all instances of the broadcast, and Scratch will take care of it automatically.

I do know that Scratch automatically deletes broadcasts, but usually I want to delete them instantly. It's a bit of a hassle to switch sprites or save and reload - an `x` button next to broadcasts will save a lot of time.

**Comment by GarboMuffin** [comment]
> Maybe we could add a feature to mass replace a broadcast in use with another one, like the feature available in the DevTools addon in SA.

This is part of the Rename broadcasts addon
